codespell_lib/data/dictionary.txt

```diff
@@ -19365,6 +19365,7 @@ intereference->interference
 intereferences->interferences
 interelated->interrelated
 interelaved->interleaved
+interepolation->interpolation
```

Comment:
```suggestion
interepolate->interpolate
interepolated->interpolated
interepolates->interpolates
interepolating->interpolating
interepolation->interpolation
```

Reply:
I did not suggest `interpolations` due to 0 hit.

* https://github.com/search?q=interepolate&type=code  11
* https://github.com/search?q=interepolated&type=code  21
* https://github.com/search?q=interepolates&type=code  5
* https://github.com/search?q=interepolating&type=code  3
* https://github.com/search?q=interepolation&type=code  178
* https://github.com/search?q=interepolations&type=code  0